frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx

```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
     const { groupsTaxonomicTypes } = useValues(groupsModel)
     const [isUsingFormulas, setIsUsingFormulas] = useState(filters.formula ? true : false)
     const lifecycles = [
-        { name: 'new', tooltip: 'Users that are new.' },
-        { name: 'resurrecting', tooltip: 'Users who were once active but became dormant, and are now active again.' },
-        { name: 'returning', tooltip: 'Users who consistently use the product.' },
-        { name: 'dormant', tooltip: 'Users who are inactive.' },
+        { name: 'new', tooltip: 'Users who signed up and did activity during period.' },
```

Comment:
```suggestion
        { name: 'new', tooltip: 'Users who both signed up and did the activity during the period .' },
```

Aside from suggestion above, how about interpolating the actual event name (e.g. "... and performed insight viewed.")?